index.js

```diff
@@ -0,0 +1,18 @@
+const hamMenu = document.querySelector('.ham-menu');
+const mobileMenu = document.querySelector('.mobile-menu');
+const closeMenu = document.querySelector('.close-icon');
+const mobileLinks = document.querySelectorAll('.mobile-link');
+
+hamMenu.addEventListener('click', () => {
+  mobileMenu.classList.toggle('active');
+});
+
+closeMenu.addEventListener('click', () => {
+  mobileMenu.classList.toggle('active');
+});
+
+mobileLinks.forEach((link) => {
+  link.addEventListener('click', () => {
+    mobileMenu.classList.toggle('active');
+  });
```

Comment:
- [ ] Your mobile hamburger displays on the desktop. Kindly fix this by displaying it as none on the desktop screen size 👍  
- This gives a more profestionl feel, as it shows you pay attention to detail 👍 

## View :
![image](https://user-images.githubusercontent.com/50990925/230916817-f1baf2aa-778a-4b01-9cab-cd79b96c6769.png)